modules/tinymce/src/core/main/ts/init/InitContentBody.ts

```diff
@@ -71,6 +71,8 @@ const mkParserSettings = (editor: Editor): DomParserSettings => {
   return removeUndefined<DomParserSettings>({
     allow_conditional_comments: getOption('allow_conditional_comments'),
     allow_html_data_urls: getOption('allow_html_data_urls'),
+    allow_mathml_attributes: getOption('extended_mathml_attributes'),
+    allow_mathml_elements: getOption('extended_mathml_elements'),
```

Comment:
This is a little bit of nitpicking, but every other parser setting matches the external option. I realise that many of them are `allow_*` and these aren't anymore, but I think keeping those matching names will help to avoid confusion in future maintenance.